### DISCUSSION TITLE
Child Theme?

### DISCUSSION THREAD

**Discussion by lestertarranza** [tailpress/tailpress#44] [Ideas]
Is a child theme possible with this setup?

**Comment by shane-hoving-pc** [comment]
Has anyone found the best way to implement a child theme, or if it's even reasonable to do so?

**Comment by jeffreyvr** [comment]
At this stage I would not create a child theme with TailPress because it is a boilerplate and kept very minimal. It's a starting point for your own custom theme.

If TailPress had more features/code, it would make more sense to use a child theme - because then you could keep up-to-date by updating the parent theme.

**Reply by shane-hoving-pc** [reply]
@jeffreyvr thank you for your insight. I appreciate the quick response.
